Arrays/Counting bits.java

```diff
@@ -0,0 +1,11 @@
+>https://leetcode.com/problems/counting-bits/
+
+class Solution {
+        public int[] countBits(int n) {
```

Comment:
Please rename file to follow camel casing and format of the code

Reply:
Sure